pokornyd/rich-text-resolver
# Portable text

### Motivation

This PR introduces portable text support to the rich text parser. Still a work in progress, portable text is currently only supported in RichTextNodeParser. Code base is going to be refactored and functionality extended to RichTextBrowserParser in later iterations. Draft PR created to provide an option to test and review existing functionality.

### Checklist


### How to test

Instantiate a RichTextNodeParser class, use the parse method on a piece of rich text from Kontent.ai. Try creating obscure edge cases to identify potential issues with the parsing logic. Check existing portable text tests for usage details.
